content/docs/1.6.0/advanced-resources/os-distro-specific/okd-support.md

```diff
@@ -0,0 +1,164 @@
+---
+title:  OCP/OKD Support
+weight: 4
+---
+
+To deploy Longhorn on a cluster provisioned with OpenShift 4.x, some additional configurations is required.
+
+- [Prerequisite](#prerequisite)
+  - [Default Path /var/lib/longhorn Setup](#default-path-varliblonghorn-setup)
+  - [Separate Path /var/mnt/longhorn Setup](#separate-path-varmntlonghorn-setup)
+    - [Create Filesystem](#create-filesystem)
+    - [Mounting Devices On Boot with MachineConfig](#mounting-devices-on-boot-with-machineconfig)
+    - [Label and Annotate Nodes](#label-and-annotate-nodes)
+- [Install Longhorn](#install-longhorn)
+- [Upgrade Longhorn](#upgrade-longhorn)
+- [Uninstall Longhorn](#uninstall-longhorn)
+  - [Uninstall Prerequisite](#uninstall-prerequisite)
+  - [Uninstalling Longhorn using kubectl](#uninstalling-longhorn-using-kubectl)
+- [Reference](#reference)
+
+## Prerequisite
+
+### Default Path /var/lib/longhorn Setup
+
+Label each node for storage with:
+
+```bash
+oc get nodes --no-headers | awk '{print $1}'
+oc label node "${NODE_NAME}" node.longhorn.io/create-default-disk=true
+```
+
+### Separate Path /var/mnt/longhorn Setup
+
+#### Create Filesystem
+
+On the storage nodes create a filesystem with the label longhorn:
+
+```bash
+oc get nodes --no-headers | awk '{print $1}'
+oc debug node/${NODE_NAME} -t -- chroot /host bash
+```
+
+Check devices are present and format them with Longhorn label:
+
+```bash
+lsblk
+sudo mkfs.ext4 -L longhorn /dev/${DEVICE_NAME}
+```
+
+#### Mounting Devices On Boot with MachineConfig
+
+The Secondary Drive needs to be mounted on every boot. Save the concents and apply the `MachineConfig` with `oc apply -f`:
+
+```yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 71-mount-storage-worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+        - name: var-mnt-longhorn.mount
+          enabled: true
+          contents: |
+            [Unit]
+            Before=local-fs.target
+            [Mount]
+            Where=/var/mnt/longhorn
+            What=/dev/disk/by-label/longhorn
+            Options=rw,relatime,discard
+            [Install]
+            WantedBy=local-fs.target
+```
+
+#### Label and Annotate Nodes
+
+Label and annotate storage nodes by:
+
+```bash
+oc get nodes --no-headers | awk '{print $1}'
+
+oc annotate node ${NODE_NAME} --overwrite node.longhorn.io/default-disks-config='[{"path":"/var/mnt/longhorn","allowScheduling":true}]'
+oc label node ${NODE_NAME} --overwrite node.longhorn.io/create-default-disk=config
+```
+
+## Install Longhorn
+
+1. Get the source code tarball
+
+```bash
+wget https://github.com/longhorn/longhorn/archive/refs/tags/v{{< current-version >}}.tar.gz
+tar xzf v{{< current-version >}}.tar.gz
+cd longhorn-{{< current-version >}}/chart/
```

Comment:
Should use the correct install solutions. You can simply refer to the install page.

Reply:
Modified to use helm command to set `openshift.enabled` true